data/field.go

```diff
@@ -160,6 +160,12 @@ func (f *Field) Append(e interface{}) {
 	f.vector.Append(e)
 }
 
+func (f *Field) AppendAll(f2 *Field) {
+	for i := range f2.vector.Len() {
+		f.Append(f2.At(i))
```

Comment:
Better to call Extend I think, then set. Not sure this a good method to have in general thought.

Maybe do this at Frame level with `Frame.Extend` and `Frame.SetRow` (and maybe add `frame.RowAt()` (like row copy but without the copy)).